libbeat/common/scan.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"bufio"
+	"bytes"
+)
+
+// ScanDelimiter return a function to split line using a custom delimiter, the delimiter
+// is stripped from the returned value.
+func ScanDelimiter(delimiter []byte) bufio.SplitFunc {
```

Comment:
I would not put functions into common which we only use in one beat. We should have at least 2 use cases.